eclair-core/src/main/scala/fr/acinq/eclair/blockchain/electrum/ElectrumWallet.scala

```diff
@@ -610,6 +606,11 @@ object ElectrumWallet {
 
     val chainHash = blockchain.chainHash
 
+    def keyStore = walletType match {
```

Comment:
Why is it a `def` and not a `val` ?

Reply:
Slipped through, it doesn't need to be a `def` so it can be a `val`